ERROR - 2023-01-20 10:48:57 --> mysqli_sql_exception: Table 'newsapp.v_access' doesn't exist in C:\NOVAL\App\mail-tracking-minerba\mail-tracking\starter\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\NOVAL\App\mail-tracking-minerba\mail-tracking\starter\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\NOVAL\App\mail-tracking-minerba\mail-tracking\starter\vendor\codeigniter4\framework\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\NOVAL\App\mail-tracking-minerba\mail-tracking\starter\vendor\codeigniter4\framework\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\NOVAL\App\mail-tracking-minerba\mail-tracking\starter\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\NOVAL\App\mail-tracking-minerba\mail-tracking\starter\vendor\codeigniter4\framework\system\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
#5 C:\NOVAL\App\mail-tracking-minerba\mail-tracking\starter\vendor\codeigniter4\framework\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#6 C:\NOVAL\App\mail-tracking-minerba\mail-tracking\starter\app\Models\ModelOtentikasi.php(17): CodeIgniter\BaseModel->first()
#7 C:\NOVAL\App\mail-tracking-minerba\mail-tracking\starter\app\Controllers\Otentikasi.php(46): App\Models\ModelOtentikasi->getEmployee('198412192009021...', '198412192009021...')
#8 C:\NOVAL\App\mail-tracking-minerba\mail-tracking\starter\vendor\codeigniter4\framework\system\CodeIgniter.php(907): App\Controllers\Otentikasi->index()
#9 C:\NOVAL\App\mail-tracking-minerba\mail-tracking\starter\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Otentikasi))
#10 C:\NOVAL\App\mail-tracking-minerba\mail-tracking\starter\vendor\codeigniter4\framework\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\NOVAL\App\mail-tracking-minerba\mail-tracking\starter\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\NOVAL\App\mail-tracking-minerba\mail-tracking\starter\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\NOVAL\\App\\ma...')
#13 {main}
CRITICAL - 2023-01-20 10:48:57 --> Table 'newsapp.v_access' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *
FROM `v_access`
WHERE `employeeId` = \'198412192009021001\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `v_access`
WHERE `employeeId` = \'198412192009021001\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `v_access`
WHERE `employeeId` = \'198412192009021001\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `v_access`
WHERE `employeeId` = :employeeId:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(250): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(622): CodeIgniter\Model->doFirst()
 7 APPPATH\Models\ModelOtentikasi.php(17): CodeIgniter\BaseModel->first()
 8 APPPATH\Controllers\Otentikasi.php(46): App\Models\ModelOtentikasi->getEmployee('198412192009021001', '198412192009021001')
 9 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\Otentikasi->index()
10 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Otentikasi))
11 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
